fetch-in-react/my-app/src/User.js

```diff
@@ -14,22 +14,23 @@ export default function User({ userId, onCancel }) {
       try {
         const response = await fetch(`https://jsonplaceholder.typicode.com/users/${userId}`);
         const jsonData = await response.json();
-        if (!ignore) {
+        if (!ignore && response.ok) {
           console.log('My jsonData:', jsonData)
           setIsLoading(false)
           return setUser(jsonData)
         }
-        throw new Error('There was an error', error)
+        throw new Error('There was an error');
```

Comment:
This will throw if `ignore` is true, which isn't an error. I'm sure you got the `ignore` code from the React reading, but it's a very strange and hard-to-understand solution (I don't fully understand how it works). I suggest you just remove it for now. In a real job you won't be using this approach for your backend fetches, anyway.